app.rb

```diff
@@ -0,0 +1,47 @@
+require './book'
+require './classroom'
+require './person'
+require './rental'
+require './student'
+require './teacher'
```

Comment:
- [ ] I think for the paths for `person`, `student` and `teacher` is wrong so since you have the files called `person_class` so you should call it the same file name so it cause me an error when running the app so kindly fix it 🔥🙏🏻😊